### ISSUE TITLE
Torrents error / permission denied on the mkdir command

### ISSUE THREAD

**Issue by onednasty** [DyonR/docker-qbittorrentvpn#46]
I'm a total newb when it comes to docker admittingly, but I've verified my UID & GID via the id command in the console.

I updated those values in the docker config for qbittorrentvpn, but I keep getting permission denied errors.

(W) 2021-06-21T00:02:41 - File error alert. Torrent: "Yellowstone.2018.S03.1080p.BluRay.x264-BORDURE". File: "/mnt/user/data/torrents/Yellowstone.2018.S03.1080p.BluRay.x264-BORDURE[rartv]/Yellowstone.2018.S03E07.1080p.BluRay.x264-BORDURE.mkv". Reason: Yellowstone.2018.S03.1080p.BluRay.x264-BORDURE[rartv] mkdir (/mnt/user/data/torrents/Yellowstone.2018.S03.1080p.BluRay.x264-BORDURE[rartv]/Yellowstone.2018.S03E07.1080p.BluRay.x264-BORDURE.mkv) error: Permission denied

I'm completely at a loss of what to do as I've verified my save path and another container created the file structure without issue. 

I can't get proxy to work on one and running into this error on this one. Please help. 

Edit: Changing the UID & GID both to zero in the settings seems to have fixed the problem. Seems odd the user I have created isn't working.

**Comment by DyonR** [comment]
Are the permissions on the 'torrents' folder set correctly with chmod?

**Comment by onednasty** [comment]
Everything seems fine. I was able to create a folder in the same directory via file explorer. 

Setting both the UID & GID to '0' in the container's settings seems to allow it to create the folder. 

Though honestly I can't even find where the files were created at this point so maybe I shouldn't have the "/mnt/user/" in the location. 

I honestly have no idea what I'm doing as I've never used Linux and just set up my UNRaid server about a week ago. 

So I don't know what chmod is, but it certainly seems the file permissions were not working for the UID/GID of the non-root user I created. 

I mean the good news is the VPN part seems to be working so thank you for your hard work in getting this created.

Would be nice if it came with VueTorrent. :P
